.golangci.yml

```diff
@@ -18,9 +18,13 @@
 # options for analysis running
 run:
   issues-exit-code: 1
-  skip-dirs-use-default: true
   modules-download-mode: readonly
   deadline: 5m
+  skip-dirs:
+    - pkg/client/clientset
+    - pkg/client/informers
+    - pkg/client/listers
+    - pkg/apis/yunikorn.apache.org/v1alpha1/zz_generated.deepcopy.go
```

Comment:
I tried this added an extra new line to the file and it did not get skipped.
Should this not be under `skip-files` (https://golangci-lint.run/usage/configuration/)?